charter.md

```diff
@@ -25,8 +25,6 @@ These metrics are expected to be comprised of:
 
 The team is responsible for maintaining [Interop 2021](https://wpt.fyi/interop-2021), [2022](https://wpt.fyi/interop-2022), [2023](https://wpt.fyi/interop-2023), and defining future interoperability efforts and metrics.
 
-If at any time the team is unable to reach consensus on which features to include in its metrics, or the set of features which has consensus is considered insufficient to fulfill the goals of the team, the effort will be paused until the situation can be resolved.
```

Comment:
Rather than remove this entirely I would prefer to say something like:

In the event that the team is unable to reach sufficient consensus to fulfill the requirements of this charter (e.g. if it is unable to agree on any metrics to publish) the effort will be paused until the situation is resolved.

Reply:
@foolip it doesn't seem like you ever did add it back in with your edits yesterday?

Reply:
I didn't, because there was no agreement on what it should say. So I'm leaving it to you :)

Reply:
This was the main part of the discussion in https://github.com/web-platform-tests/interop/issues/329.

That said:

Most obviously, existing metrics will likely still need maintenance, even if we reach a point of deciding not to publish future ones.

It's also unclear what it means for it to be "paused". 

The motivation to have some text here I think is two-fold:

1. If at some point the team believes there is no need to publish further, future metrics (or at least not on an annual cadence), or
2. If at some point the team is unable to reach consensus on what a new metric should be.

In the former case I don't think we necessarily need to do anything (maybe maintenance could be handed over to the Core Team, but that's not necessarily the case).

In the latter case we could outline various processes, but I don't know it's worthwhile, and it's entirely plausible the group could still functionally maintain existing metrics.

Reply:
@jgraham do you still want to keep some text here? I'm fine either way.

Reply:
I think it makes sense to say _something_ about what happens if we are unable to fulfill the requirements of the charter (e.g. because we can't reach consensus on continuing to produce metrics). I don't think it makes sense to try to enumerate every possible case.

Per @gsnedders points, I think "we continue to maintain existing Interop-style annual metrics but can't agree to publish new ones" would count as no longer fulfilling the requirements of the charter. I think the case where we do reach consensus, but the consensus is "we don't want to publish more metrics" can be understood as the group deciding to disband (that is, I don't think routine maintenance alone is enough to warrant the group existing). And I think the case where we settle on a single ongoing metric but with periodic updates to its composition counts as the group still operating as designed.

Reply:
https://github.com/web-platform-tests/interop/pull/342 contains a proposal here.

Reply:
And that has now landed.